chain/client-primitives/src/debug.rs

```diff
@@ -47,17 +47,33 @@ pub struct DebugChunkStatus {
 #[derive(Serialize, Deserialize, Debug)]
 pub struct DebugBlockStatus {
     pub block_hash: CryptoHash,
+    pub prev_block_hash: CryptoHash,
     pub block_height: u64,
+    pub block_timestamp: u64,
     pub block_producer: Option<AccountId>,
+    pub is_block_missing: bool, // only header available
+    pub is_on_canonical_chain: bool,
     pub chunks: Vec<DebugChunkStatus>,
     // Time that was spent processing a given block.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub processing_time_ms: Option<u64>,
-    // Time between this block and the next one in chain.
-    pub timestamp_delta: u64,
```

Comment:
Is it intentional that we removed this?

Reply:
Yeah, it's computed on the UI side now, and displayed as "block delay", computed as timestamp minus parent's timestamp.